src/components/Menu/ControlledMenu.js

```diff
@@ -0,0 +1,174 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import omit from 'lodash/omit';
+import { getUniqueHash } from '../../utils';
+import MenuItem from './MenuItem';
+
+const propTypes = {
```

Comment:
Why did you specify the `propTypes` outside of the class instead of `static propTypes = { ... }`; You could later reference them like `ControlledMenu.propTypes` if you need to.

Reply:
I believe it's because we use [transform-react-remove-prop-types](https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types).

Reply:
@philippfrank @brunottonurb is right here. If you need a reference to the propType object, you need to persist it outside `Component.propTypes` since those will be stripped in production builds